src/com/jwetherell/algorithms/mathematics/Division.java

```diff
@@ -3,6 +3,11 @@
 public class Division {
 
     public static final long division(int a, int b) {
+        
```

Comment:
The method will throw an exception by ensuring the value of b before the result "long" is calculated avoiding unexpected results.